<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_person</class>
 <widget class="QDialog" name="Add_person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加学生</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);
background-color: rgb(170, 255, 255);</string>
     </property>
     <property name="text">
      <string>添加员工</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>工号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id_acc">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>请输入工号</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_acc">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>请输入姓名</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>基本工资：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="salary">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>请输入基本工资</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>岗位津贴：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="allowance">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>请输入岗位津贴</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>医疗保险：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>请输入医疗保险</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
